go.mod

```diff
@@ -9,5 +9,8 @@ require (
 	github.com/filecoin-project/lotus v0.7.1
 	github.com/filecoin-project/specs-actors v0.9.10
 	github.com/minio/blake2b-simd v0.0.0-20160723061019-3f5f724cb5b1
+	github.com/stretchr/testify v1.6.1
 	github.com/whyrusleeping/cbor-gen v0.0.0-20200826160007-0b9f6c5fb163
 )
+
+replace github.com/filecoin-project/go-address => ../go-address
```

Comment:
waiting on this [PR](https://github.com/filecoin-project/go-address/pull/14) to land before removing

Reply:
great!

Reply:
Claim my asset